swagger.yaml

```diff
@@ -844,4 +1005,54 @@ components:
         lastUpdatedAt:
           type: string
           format: date-time
-          description: Date and time when the software entry was last updated
\ No newline at end of file
+          description: Date and time when the software entry was last updated
+        
+    Questionnaire:
+      type: object
+      properties:
+        id:
+          type: integer
+          description: Unique identifier for the questionnaire
+        title:
+          type: string
+          description: Title of the questionnaire
+        description:
+          type: string
+          description: Description of the questionnaire
+        options:
+          type: array
+          items:
+            $ref: "#/components/schemas/Questions"
+          description: List of questions in the questionnaire
+        tags:
+          type: array
+          items:
+            type: string
+          description: List of tags for the questionnaire
+        passedUsers:
+          type: array
+          items:
+            type: integer
+          description: List of users who have passed the questionnaire
+        passScore:
+          type: integer
+          description: Minimum score required to pass the questionnaire
+
+    Questions:
+      type: object
+      properties:
+        question:
+          type: string
+          description: The question text
+        options:
+          type: array
+          items:
+            type: object
+            properties:
+              label:
+                type: string
+                description: Display text for the option
+              value:
+                type: boolean
+                description: Boolean value of the option
+          description: List of correct answers with labels and boolean values
```

Comment:
From what I understand this description is not correct- it's a list of answers,which includes the correct answer, not a list of correct answers.

Something like:
List of options objects containing options labels and answer status boolean